saleor/account/models.py

```diff
@@ -139,6 +141,15 @@ class Meta:
                     'Permission description', 'Impersonate customers.')))
 
     def get_full_name(self):
+        if self.first_name or self.last_name:
+            not_empty_names = filter(None, [self.first_name, self.last_name])
```

Comment:
I was wondering if we could simplify the logic here and return the "full name" only if we have both first name and last name (the same in case of getting them from the address). Otherwise we would return just an email. E.g. if we had only "John" or just "Doe" somewhere in the UI as the user representation - does it provide any useful value? In that case, "john.doe@example.com" feels more valuable.

Reply:
Not related to my previous comment, but generally we rather don't use `filter` in our Python code. Here we have only (and always) two list items to check so we might as well write something like: `('%s %s' % (first_name, last_name)).strip()` to get the same result. In case a list contains more elements, a more idiomatic way to do the filtering is by using list comprehensions.

Reply:
What if a person has only one name?